.github/ISSUE_TEMPLATE/performance.yml

```diff
@@ -0,0 +1,136 @@
+name: Performance Issue Report
+description: This report is for the performance-related issue
+title: "[Performance]: "
+labels: ["performance", "support_request"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Please provide all the necessary information to expedite the response.
+  - type: input
+    id: ov_version
+    attributes:
+      label: OpenVINO Version 
+      description: OpenVINO version, branch, or tag in OpenVINO GitHub
+      placeholder: 2021.4.0 LTS / Master Branch / tag 2022.3.0
+    validations:
+      required: false
+  - type: input
+    id: target_platform
+    attributes:
+      label: Target Platform
+      description: The platform that the OpenVINO source is built on
+      placeholder: 11th Gen Intel Core i7-1185G7
+    validations:
+      required: false
+  - type: dropdown
+    id: os
+    attributes:
+      label: Operating System
+      description: What OS are you using?
+      options:
```

Comment:
Please add virtual machine information like wsl2 on Windows

Reply:
add another line for "WSL2 for windows"